control/controlhttp/constants.go

```diff
@@ -18,3 +28,51 @@ const (
 	// to do the protocol switch is located.
 	serverUpgradePath = "/ts2021"
 )
+
+// Dialer contains configuration on how to dial the Tailscale control server.
+type Dialer struct {
+	// Host is the host to connect to.
```

Comment:
is the host name? (I assume it must be a DNS name and not a raw IP?)

and no port number, since that's below.

Reply:
Renamed to `Hostname` to be more clear; added comment stating that this should not contain a port.